src/v2/guide/components-slots.md

```diff
@@ -85,145 +163,396 @@ Para estos casos, el elemento `<slot>` tiene un atributo especial, `name`, el cu
+
+```html
 <base-layout>
-  <h1 slot="header">Aquí podría haber un título de página.</h1>
+  <template v-slot:header>
+    <h1>Aquí podría ir un título de página</h1>
+  </template>
 
-  <p>Un párrafo para el contenido principal.</p>
-  <p>Y otro más.</p>
+  <template v-slot:por defecto>
+    <p>Un párrafo para el contenido principal.</p>
+    <p>Y otra más. </p>
+  </template>
 
-  <p slot="footer">Aquí hay alguna información de contacto</p>
+  <template v-slot:footer>
+    <p>Aquí va alguna información de contacto</p>
+  </template>
 </base-layout>
 ```
 
-Todavía puede haber un slot sin nombre, que es el **slot predeterminado** que sirve como una salida para todos los contenidos no coincidentes. En los dos ejemplos anteriores, el HTML representado sería:
+De cualquier manera, el HTML renderizado será:
 
-``` html
+```html
 <div class="container">
   <header>
-    <h1>Aquí podría haber un título de página.</h1>
+    <h1>Aquí podría ir un título de página</h1>
   </header>
   <main>
     <p>Un párrafo para el contenido principal.</p>
-    <p>Y otro más.</p>
+    <p>Y otra más.</p>
   </main>
   <footer>
-    <p>Aquí hay alguna información de contacto</p>
+    <p>Aquí va alguna información de contacto</p>
   </footer>
 </div>
 ```
 
-## Contenido del slot por defecto
+Note que **`v-slot` sólo puede ser agregado a un `<template>`** (con [una excepción](#Abbreviated-Syntax-for-Lone-Default-Slots)), a diferencia del atributo depreciado [`slot`](#Deprecated-Syntax).
 
-Hay casos en los que es útil proporcionar un slot con contenido predeterminado. Por ejemplo, un componente `<submit-button>` puede querer que el contenido del botón sea "Enviar" de forma predeterminada, pero también permite a los usuarios anular el contenido con "Guardar", "Cargar" o cualquier otra cosa.
+## Slots con Scope
+
+> Actualizado en 2.6.0+. [Ver aquí](#Sintaxis-Obsoleta) para la sintaxis obsoleta usando el atributo "slot-scope".
 
-Para lograr esto, especifique el contenido predeterminado entre las etiquetas `<slot>` en la plantilla de su componente:
+A veces, es útil para el contenido del slot tener acceso a los datos sólo disponibles en el componente hijo. Por ejemplo, imagine un componente `<current-user>` con la siguiente plantilla:
 
 ```html
-<button type="submit">
-  <slot>Enviar</slot>
-</button>
+<span>
+  <slot>{{ user.lastName }}</slot>
+</span>
 ```
 
-Si el contenido del slot es proporcionada por el padre, reemplazará el contenido predeterminado.
+Podríamos querer reemplazar este contenido alternativo para mostrar el nombre del usuario, en lugar del apellido, así:
 
-## Scope de compilación
+```html
+<current-user>
+  {{ user.firstName }}
+</current-user>
+```
 
-Cuando desee utilizar datos dentro de un slot, como en:
+Eso no funcionará, sin embargo, porque solo el componente `<current-user>` tiene acceso al `user` y el contenido que estamos proporcionando se ha renderizado en el padre.
```

Comment:
This phrase looks a bit weird, can you double check it?

Reply:
Yes is true. Even in English was a little tricky. I will try to explain better as the translation seems to be a bit confuse

Reply:
Were you able to take a look at this phrase?

Reply:
Let's see if now is better :)

Reply:
Way better 😄